### PR TITLE
Responsive Design -- Tnw clone

### PR DESCRIPTION
**Pull request by keddo**
**Building with Responsive Design**

This project, Consists of  building a responsive website. By cloning The Next Web, a tech-focused magazine.

I am kindly requesting for review.
Thank you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,14 +6,1394 @@
...
+                </div>
+                <a class="cover-trigger"
+                    href="https://thenextweb.com/gaming/2020/05/08/detroit-lions-use-animal-crossing-reveal-2020-schedule/"
+                    data-event-category="Homepage" data-event-action="Top story"
+                    data-event-label="Detroit Lions use Animal Crossing (yes, really) to reveal 2020 schedule"
+                    data-event-non-interaction="true"></a>
+                <div class="cover-gradient">
+
+                </div>
+                <div class="cover-logo"></div>
```
Comment (lucasmazo32) [inline index.html]: The logo crops when I am reducing the size of the page. Kindly fix that.

Screenshot from your page. 

image

File: index.html [hunk 2]
```diff
@@ -6,14 +6,1394 @@
...
+                <a class="cover-trigger" href="https://thenextweb.com/events/rebrand" target="_blank"
+                    rel="noopener noreferrer" data-event-category="Homepage" data-event-action="Top story"
+                    data-event-label="Headspace, Airbnb &amp; Google share brand and digital marketing strategy at Re:Brand online event"
+                    data-event-non-interaction="true"></a>
+                <header class="cover-header">
+                    <h2 class="cover-title">
+                        <a href="#">
+                            Headspace, Airbnb & Google share brand and digital marketing strategy at Re:Brand online
+                            event
+                        </a>
```
Comment (lucasmazo32) [inline index.html]: The letters on the side articles are cropped when reducing the size. Kindly fix that.

> Screenshot from your page.
image

File: index.html [hunk 3]
```diff
@@ -6,14 +6,1394 @@
...
+                        <li>
+                            <a href="#" class="deals-header-more">See more</a>
+                        </li>
+                        <li>
+                            <a href="#" class="deals-header-more">Get updates by email</a>
+                        </li>
+                        <li>
+                            <a href="#" class="deals-header-more">Add info</a>
+                        </li>
+                    </ul>
```
Comment (lucasmazo32) [inline index.html]: When there are only three articles, the text to the side of latest funding should be below.

> Original page
image

>Your page
image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

You are really close, please implement the changes below for your project to get approved!

### Happy coding! :computer:

## Lucas Mazo

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required!
Hi @keddo, well done on this project so far! However, there are some requirements you need to implement to complete. These are:
## Required:
- [x] Fix [HTML validator](https://validator.w3.org/#validate_by_upload) errors.
- [x] Avoid using inline CSS.
- [x] The logo isn't proportionally sized for large screens.
- [x] Latest news section heading is partially hidden under the top 3 articles section on large screens. Please fix this.
- [x] There should be no small breakpoints. The base styles should be enough for the mobile screen.([screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))

## Optional:
- [ ] It is redundant to give the `<main> tag` a class of 'main', and the `<footer> tag` a class of 'footer'. It is ok to use tag selectors in this case. I commend you on the use of semantic HTML. :+1: 

__Kindly work on the required changes and submit another code review request__
_Feel free to contact me on slack about any inquiries regarding this review_ :wink: 
__Treasure Kabareebe__

**Comment by keddo** [comment]
**Changes made to the above Review:**
* HTML validation errors got fixed
* Inline CSS removed
* Logo fixed to responsive
* Issues related to latest news section fixed
* base style for media queries add.

**Kindly requesting for review.
Thanks**

**Review by lucasmazo32 — APPROVED** [review]
# Project approved :rocket:

#### Great job!

#### Your project looks great!

### You can go ahead and merge this pull request!

### Happy coding! :computer:

## Lucas Mazo

**Pull request merged**